lib/generic/modules/allergic_rhinitis.json

```diff
@@ -0,0 +1,351 @@
+            "allergens like pet dander, mold, and dust mites."
+          ]
+        },
+        {
+          "distribution": 0.4,
+          "transition": "Has_Perennial_And_Seasonal_Allergic_Rhinitis",
+          "remarks": [
+            "Perennial allergies flare up in the spring with additional allergies, like grass and ",
+            "tree pollens."
+          ]
+        }
+      ]
+    },
+
+    "Has_Seasonal_Allergic_Rhinitis": {
+      "type": "ConditionOnset",
+      "assign_to_attribute": "has_allergic_rhinitis",
+      "target_encounter": "Allergic_Rhinitis_Diagnosis",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": "367498001",
+          "display": "Seasonal allergic rhinitis"
+        }
+      ],
+      "direct_transition": "Allergic_Rhinitis_Diagnosis"
+    },
+
+    "Has_Perennial_Allergic_Rhinitis": {
+      "type": "ConditionOnset",
+      "assign_to_attribute": "has_allergic_rhinitis",
+      "target_encounter": "Allergic_Rhinitis_Diagnosis",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": "429195002",
+          "display": "Allergic rhinitis caused by animal dander"
+        },
+        {
+          "system": "SNOMED-CT",
+          "code": "449729000",
+          "display": "Allergic rhinitis caused by house dust mite"
+        }
+      ],
+      "remarks": ["These people do not have seasonal hay fever like the other cases.",
+                  "These allergies are mainly to pet dander and dust."],
+      "direct_transition": "Allergic_Rhinitis_Diagnosis"
+    },
+
+    "Has_Perennial_And_Seasonal_Allergic_Rhinitis": {
+      "type": "ConditionOnset",
+      "assign_to_attribute": "has_allergic_rhinitis",
+      "target_encounter": "Allergic_Rhinitis_Diagnosis",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": "232353008",
+          "display": "Perennial allergic rhinitis with seasonal variation"
+        }
+      ],
+      "direct_transition": "Allergic_Rhinitis_Diagnosis"
+    },
+
+    "Allergic_Rhinitis_Diagnosis": {
+      "type": "Encounter",
+      "class": "outpatient",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": "185345009",
+          "display": "Encounter for symptom"
+        }
+      ],
+      "direct_transition": "Prescribe_OTC_Medication"
+    },
+
+    "Prescribe_OTC_Medication": {
+      "type": "MedicationOrder",
+      "target_encounter": "Allergic_Rhinitis_Diagnosis",
+      "remarks": [
+        "There are several antihistamines and delivery methods to choose from. Different ones work for each person.",
+        "For variety I listed oral (Claritin, and Benadryl), nasal (Flonase), and decongestants (Sudafed) medicines, ",
+        "but used the generic medicine names and dosages. Most doctors would likely recommend any combination of these."
+      ],
+      "codes": [
+        {
+          "system": "RxNorm",
```

Comment:
Adding multiple codes is for representing the same concept in different systems, ex SNOMED and ICD-10 for a condition. In this case adding 4 codes isn't going to add multiple different medications, it's going to add one medication with 4 different codes. If we want 4 different medications prescribed it would have to be 4 separate MedicationOrder states